crates/store/re_protos/src/v0/rerun.common.v0.rs

```diff
@@ -11,18 +11,48 @@ pub struct RerunChunk {
     #[prost(bytes = "vec", tag = "1000")]
     pub payload: ::prost::alloc::vec::Vec<u8>,
 }
+impl ::prost::Name for RerunChunk {
```

Comment:
This file too should be marked `linguist-generated=true`. Maybe just mark the whole folder?